setup.py

```diff
@@ -138,7 +138,8 @@ def main():
                      'requests',
                      'defusedxml',
                      'dugong >= 3.4',
-                     'llfuse >= 0.39' ]
+                     'llfuse >= 0.39',
+                     'google-api-python-client >= 1.4.2', ]
```

Comment:
Is the API really that complicated that a separate module is needed? There is a good reason why we do not e.g. use the `boto` module for Google Storage.